http-api/src/commit.rs

```diff
@@ -16,16 +16,16 @@ pub struct CommitsQueryString {
 
 #[derive(Serialize)]
 pub struct CommitTeaser {
-    pub header: Header,
+    pub header: git::Commit,
     pub context: CommitContext,
 }
 
 #[derive(Serialize)]
-pub struct Commit {
-    pub header: Header,
-    pub stats: radicle_source::commit::Stats,
-    pub diff: radicle_surf::diff::Diff,
-    pub branches: Vec<radicle_source::Branch>,
+pub struct Header {
```

Comment:
Since `git::Commit` is qualified, we can call this simply `Commit`, as it includes more than just the header.